apps/src/applab/applab.js

```diff
@@ -894,10 +899,16 @@ Applab.init = function(config) {
     isEmbedView: !!config.embed,
     isReadOnlyWorkspace: !!config.readonlyWorkspace,
     isShareView: !!config.share,
-    isViewDataButtonHidden: !!config.level.hideViewDataButton
+    isViewDataButtonHidden: !!config.level.hideViewDataButton,
+    instructionsMarkdown: studioApp.getMarkdownInstructions(config.level)
```

Comment:
It looks like this is the only use of `getMarkdownInstructions`.  Could we simplify the contract of that method by having it always return raw markdown and doing the processing in the rendering component? (I expect the `instructionsMarkdown` property to contain markdown anyway, not rendered markup.) To do this cleanly, we might want to get `marked` via browserify(-global?)-shim instead of adding a reference to window in a React component.